lib/elixir/lib/process.ex

```diff
@@ -399,8 +398,8 @@ defmodule Process do
 
   Inlined by the compiler.
   """
-  @spec demonitor(reference) :: true
   @spec demonitor(reference, options :: [:flush | :info]) :: boolean
+  @spec demonitor(reference, []) :: true
```

Comment:
By clear I meant easy to read and understand, rather than explicit. It is awkward to have multiple specs for different list elements because it is not `[]` that means the return value is `true` but the absence of `:info` from the list. For example `[:flush]` is always going to be `true` The spec is not showing the relationship. We should only use multiple specs for the same arity when multiple specs help show the cause of the differences.